.github/workflows/ci.yml

```diff
@@ -78,6 +78,8 @@ jobs:
 
     - name: Try installing packages
       run: aiida-registry test-install
+    - name: Generate entrypoints data
```

Comment:
I suggest you simply modify the previous step instead.
I see below that you are installing all the plugins again here (which is very slow) - we should only do this once